<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh boundary layer settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>3</width>
       <height>3</height>
      </size>
     </property>
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Lab_FinestSize">
        <property name="text">
         <string>Finest size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Gui::InputField" name="FinestSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="min_thickness" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="decimals" stdset="0">
         <number>2</number>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Lab_CoarsestSize">
        <property name="text">
         <string>Coarsest size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Gui::InputField" name="CoarsestSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="min_thickness" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="decimals" stdset="0">
         <number>2</number>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Lab_FinestThickness">
        <property name="text">
         <string>Finest thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Gui::InputField" name="FinestThickness">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="min_thickness" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="decimals" stdset="0">
         <number>2</number>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Lab_CoarsestThickness">
        <property name="text">
         <string>Coarsest thickness</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Gui::InputField" name="CoarsestThickness">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="min_thickness" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="decimals" stdset="0">
         <number>2</number>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
